specification/botservice/resource-manager/Microsoft.BotService/preview/2018-07-12/examples/CreateEnterpriseChannel.json

```diff
@@ -0,0 +1,77 @@
+{
+  "parameters": {
+    "subscriptionId": "subscription-id",
+    "resourceGroupName": "OneResourceGroupName",
+    "api-version": "2018-07-12",
+    "resourceName": "contoso-dl",
+    "parameters": {
+      "location": "West US",
+      "sku": {
+        "name": "S1"
+      },
+      "etag": "etag1",
+      "tags": {
+        "tag1": "value1",
+        "tag2": "value2"
+      },
+      "id": "someid",
+      "properties": {
+        "nodes": [
+          {
+            "id": "00000000-0000-0000-0000-000000000000",
```

Comment:
How do users know what to put here? I'm curious.

Reply:
It has to be a guid. I could allow empty on create calls and generate one for them.

After the create they will have one of course - the get will return it.